src/styles/blocks/stop-watch.scss

```diff
@@ -0,0 +1,56 @@
+.stop-watch {
+  @include circle($watch-size);
+  position: relative;
+  display: flex;
+  justify-content: center;
+
+  border: 5px solid $color-main-accent;
+
+  &__center {
+    @include circle($center-size);
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+
+    background-color: $color-main-accent;
+    z-index: 100;
+  }
+
+  &__minute-hand {
+    width: $hand-width;
+    height: 30%;
+
+    @include hand-position;
+
+    border-radius: $hand-radius;
+    background-color: #f75f28;
+    z-index: 50;
+
+    animation: go-around 3600s linear infinite;
+  }
+
+  &__second-hand {
+    width: $hand-width;
+    height: 45%;
+
+    @include hand-position;
+
+    border-radius: $hand-radius;
+    background-color: #6495ed;
+
+    animation: go-around 60s linear infinite;
```

Comment:
As for me, u can create mixin for this value

@mixin name ($duration) {
 animation: go-around $duration linear infinite;
}

Its up to u)

Reply:
Yea, it is really good advice. DRY

Reply:
Thank you so much for your advice and for checking my work! I'll fix it now